src/umleditor/mvc_model/help_command.py

```diff
@@ -43,6 +43,7 @@ def help_menu():
         #Relation Commands
         "Relation Commands:\n\t"
         "rel -a 'src' 'dest' 'type' - adds a relationship between class 'src' and class 'dest' of type 'type'\n\t"
+        "Relationship types: 'aggregation', 'composition', 'inheritance', 'realization'\n\t"
```

Comment:
add a \t to the start of this maybe? That way it sticks out a bit more as being a continuation of the above line?

Do we want to abbreviate these instead of having the full word?